content/en/docs/tasks/security/authorization/rbac-groups/index.md

```diff
@@ -3,6 +3,8 @@ title: Authorization for groups and list claims
 description: Tutorial on how to configure the groups-base authorization and configure the authorization of list-typed claims in Istio.
 weight: 4
```

Comment:
Could you change the weight to 30 so that this page is after the HTTP and TCP task?